### PR TITLE
cleaned up unused console.logs, added search button fucntionality

### PR DESCRIPTION
**Pull request by EddieJorden**

### CODE REVIEW

File: src/components/searchbar/SearchBar.jsx [hunk 1]
```diff
@@ -2,26 +2,30 @@ import { updateUserInput } from './searchBarSlice';
...
 import { updateSearchTerm } from "../reddit/RedditSlice";
 import { selectUserInput } from "./searchBarSlice";
+import React from 'react';
 
 const SearchBar = () => {
-    const dispatch = useDispatch()
-    const userInput = useSelector(selectUserInput)
+    const dispatch = useDispatch();
+    const userInput = useSelector(selectUserInput);
+    const [value, setValue] = React.useState('');
```
Comment (vevo-var-nazari) [inline src/components/searchbar/SearchBar.jsx]: you already have userInput available here. you don't need value as a separate state to check

**Review by vevo-var-nazari — CHANGES_REQUESTED** [review]
Remove new state variable and use existing variable to toggle disabled

**Review by vevo-var-nazari — CHANGES_REQUESTED** [review]
.

File: src/components/searchbar/SearchBar.jsx (hunk 1 above)
Comment (EddieJorden) [reply src/components/searchbar/SearchBar.jsx]: got it

**Review by VarTheVar — APPROVED** [review]

**Pull request merged**